persistence-modules/spring-hibernate-5/src/main/java/com/baeldung/hibernate/onetoone/HibernateUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.hibernate.onetoone;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.service.ServiceRegistry;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory;
+
+    private static SessionFactory buildSessionFactory(Strategy strategy) {
+        try {
+            // Create the SessionFactory from hibernate-annotation.cfg.xml
+            Configuration configuration = new Configuration();
+
+            for (Class<?> entityClass : strategy.getEntityClasses()) {
+                configuration.addAnnotatedClass(entityClass);
+            }
+            configuration.configure("onetoone.cfg.xml");
```

Comment:
Would you do me a huge favor and rename this as "one-to-one.cfg.xml"? This will make it easier to read for our readers.